devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/ImageTask.java

```diff
@@ -1,16 +1,29 @@
 
 package io.quarkus.gradle.tasks;
 
+import java.util.Collections;
 import java.util.Map;
 
+import org.gradle.api.tasks.options.Option;
+
 public abstract class ImageTask extends QuarkusBuildProviderTask {
 
+    boolean dryRun;
+
+    @Option(option = "dry-run", description = "The container image extension to use for building the image (e.g. docker, jib, buildpack, openshift).")
```

Comment:
Description is incorrect. Steal the text from the CLI option?

https://github.com/quarkusio/quarkus/blob/main/devtools/cli/src/main/java/io/quarkus/cli/common/RunModeOption.java